modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * metering/configuring-metering/metering-configure-reporting-operator.adoc
+
+[id="metering-exposing-the-reporting-api_{context}"]
+= Exposing the reporting API
+
+On {poduct-title} the default metering installation automatically exposes a Route making the reporting API available. This provides the following features:
+
+* Automatic DNS
+* Automatic TLS based on the cluster CA
+
+As well, the default installation makes it possible to take advantage of the Openshift service serving certificates to protect the reporting API with TLS. The Openshift OAuth proxy is deployed as a side-car container for `reporting-operator`, which protects the reporting API with authentication.
+
+[id="metering-openshift-authentication_{context}"]
+== Using Openshift Authentication
+By default, the reporting API is secured with TLS and authentication. This is done by configuring the `reporting-operator` to deploy a pod containing both the `reporting-operator's` container, and a sidecar container running Openshift auth-proxy.
+
+In order to access the reporting API, the metering operator exposes a route. Once that route has been installed, you can run the command below to get the route's hostname.
```

Comment:
Since this is a procedure, I suggest formatting this as numbered steps if possible